chsdi/tests/integration/test_owschecker.py

```diff
@@ -15,6 +17,25 @@ def tearDown(self):
         super(TestOwsChecker, self).tearDown()
         del self.capture
 
+    def _callFUT(self):
+        base_url = 'http://wms.geo.admin.ch'
```

Comment:
What do you mean? should I add at production.ini something like: base_url = ${base_url}
or
use here something like this: base_url = "http://" + app.registry.settings['wmshost'] + '/'